docs/topics/vscode-plugin-identify-resolve-issues.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * docs/vscode-plugin-guide/master.adoc
+
+[id='vscode-plugin-identify-resolve-issues_{context}']
+= Analyzing your projects with the {VscodePluginName}
+
+.Prerequisites
+
+* Install `mta-cli`. s
+
+.Procedure
+
+. Launch {VscodePluginName} and click the *{ProductName}* icon in the left panel.
++ *{ProductName}* appears at the top of the *Explorer* pane of the screen, below the menu bar.
+. To create a new configuration, click *+* next to *{ProductName}* .
++ The configuration options open. *{ProductName}* gives the configuration a default name.
+. Optional: Enter a new  *Name*.
+. To select the application to test, in the *Input* section of the interface, click *Add* and select the application to test.
+. Select one or more *target migration paths*.
+. To add a target migration path, click *Add,* enter the desired path, and click *Enter.*
+
+. Review the following *Advanced options* and make any necessary changes:
+
```

Comment:
I think we can remove this section. Didn't you say that the UI has tooltips and explanations?